src/uwtools/resources/rocoto.jsonschema

```diff
@@ -0,0 +1,102 @@
+{
+    "$defs": {
+        "task": {
+            "type": "object",
+            "properties": {
+                "attrs": {
+                    "type": "object",
+                    "properties": {
+                        "name": {
+                            "type": "object"
+                        },
+                        "maxtrices": {
```

Comment:
`maxtries`?

At some point -- maybe sooner rather than later -- I think we should have a representative YAML file in this repo and either have a procedure for testing it against this schema from the command line with `uw config validate`, or via a unit test. Otherwise, it's going to be hard to judge the correctness of the schema, and we may not notice typos. I think we probably need to co-evolve the schema and a representative YAML file.

Reply:
I could add some from the SRW tests. Any idea how many we might want? Would just one be a good enough place to start?

Reply:
One would be infinitely better than zero.

Reply:
One test would definitely be a great place to start. I also see in all the documentation that `maxtries` is actually the correct spelling. This should be updated now in the latest push.